ci/rpm7/build.sh

```diff
@@ -57,7 +57,8 @@ function _usage()
 function _fix_tests()
 {
     # This function adjust functional tests to reduce time, that is spent on the tests.
-    # TODO: this function created to reduce the time that is spent on functional tests because of travis time limits. Should be fixed.
+    # TODO: this function created to reduce the time that is spent on functional tests (originally due to a travis time limit
+    # that I'm not sure if stills in GitHub Actions). Should be fixed.
```

Comment:
@wistefan do you know if there any time limit in GitHub Actions?

Maybe this TODO can be now addressed and this somehow dirty hack to be removed :) In that case, section "Changes in tests" in ci/README.md should be also fixed.

Reply:
I don't know of such a limit.

Reply:
So let's try to remove the hack (commit 9e0f6c0684cc0806fdf24167552726043651c357) and see if the CI works fine. It would be easy to revert in the case it fails.

Reply:
It worked (https://github.com/telefonicaid/fiware-orion/pull/3755/checks?check_run_id=1723798190):

![imagen](https://user-images.githubusercontent.com/1534240/105003397-e3b3ad00-5a32-11eb-96af-ccc57018bd0b.png)